### PR TITLE
tweaks techwebs

### PR DESCRIPTION
**Pull request by Dorsisdwarf**
1) Rearranges the nodes so that they're organised a bit better
2) Fixes some of the nodes being named the same thing as other nodes
3) Makes ayytech have some requirements, increases first tech cost to 5k
4) Adds extra requirements to stuff like flightsuits and combat implants because they were too easy.

:cl: Dorsisdwarf
tweak: Minor fixes to some techweb nodes
balance: Made flight suits, combat implants, and combat modules require more nodes

/:cl:

### CODE REVIEW

**Comment by Repukan** [comment]
There's a huge gap between "a little more expensive" and doubling.

**Comment by XDTM** [comment]
also between doubling and quadrupling

**Comment by Dorsisdwarf** [comment]
I meant the fact it needs actual tech to get to but I guess that wasnt so clear

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -366,40 +413,11 @@
 	id = "combat_cyber_implants"
 	display_name = "Combat Cybernetic Implants"
 	description = "Military grade combat implants to improve performance."
-	prereq_ids = list("adv_cyber_implants")	//Needs way more reqs.
+	prereq_ids = list("adv_cyber_implants","weaponry","NVGtech","high_efficiency")	//Needs way more reqs.
```
Comment (DoctoreMad) [inline code/modules/research/techweb/all_nodes.dm]: Remove the comment for more reqs while you're here.

File: code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 //Current rate: 132500 research points in 90 minutes
```
Comment (kevinz000) [inline code/modules/research/techweb/all_nodes.dm]: please make sure this is calculated to still be 90 minutes with only passive point generation and such.
as in, calculate how many points every single non-hidden unlockable node is, and then do the necessary math and set the generation rate in research subsystem to match.

Comment (DoctoreMad) [reply code/modules/research/techweb/all_nodes.dm]: >and set the generation rate in research subsystem to match.

Alternatively, just delete this PR.

Comment (Dorsisdwarf) [reply code/modules/research/techweb/all_nodes.dm]: There's only a net difference of 500 research points if I'm not mistaken. Is a difference of, what, 0.3% worth poking the algorithm for?

Comment (Arianya) [reply code/modules/research/techweb/all_nodes.dm]: I mean, the comment needs updating if the rate is changing

Comment (ninjanomnom) [reply code/modules/research/techweb/all_nodes.dm]: Forcing it to always be 90 minutes seems unnecessary to me

**Review by ninjanomnom — APPROVED** [review]

File: code/modules/research/techweb/all_nodes.dm (hunk 2 above)
Comment (Shadowlight213) [reply code/modules/research/techweb/all_nodes.dm]: 90 minute overall seems like a bad design. You could subvert this by just having a not very useful research node be super expensive, justifying raising the rate, and unlocking everything else faster.

Comment (Shadowlight213) [reply code/modules/research/techweb/all_nodes.dm]: 90 minute overall seems like a bad design. You could subvert this by just having a not very useful research node be super expensive, justifying raising the rate, and unlocking everything else faster.

Comment (Jalleo) [reply code/modules/research/techweb/all_nodes.dm]: I agree with Shadowlight.

**Pull request merged**